client/src/main/java/module-info.java

```diff
@@ -1,20 +1,23 @@
-/** */
 module com.pk {
-  requires transitive javafx.controls;
-  requires javafx.fxml;
-  requires static lombok;
-  requires org.slf4j;
-  requires java.sql;
-
-  opens com.pk.lanserver;
-  opens com.pk;
-  opens com.pk.frontend.menu;
-
-  exports com.pk;
-  exports com.pk.logic;
-  exports com.pk.logic.exceptions;
-  exports com.pk.lanserver;
-  exports com.pk.lanserver.exceptions;
-  exports com.pk.lanserver.models;
-  exports com.pk.frontend.menu;
-}
+    requires javafx.controls;
+    requires javafx.graphics;
+    requires javafx.fxml;
+    requires javafx.swing;
+    requires javafx.media;
```

Comment:
Are you sure that media and swing are required? I am able to run it without them.